docs/configuration/batch-users.md

```diff
@@ -64,4 +64,21 @@ You can run the playbook and reference that extra file:
 ```bash
 ansible-playbook student-create.yml -u ubuntu -e "studentdef=students.csv" -e @students-config.yml
 ```
+````
+
+
+## Running the playbook to delete users
```

Comment:
Should we remove the mention of `student-remove.yml` above if this is now in its own section?

![image](https://user-images.githubusercontent.com/591645/162558403-efe4e715-2aae-4a73-b0f5-dab67d496b82.png)

Reply:
Oups. Sorry. Forgot to remove this.